MonMaraicher/DetailFarmerView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailFarmerView.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 27/02/2024.
+//
+
+import SwiftUI
+import MapKit
+
+struct DetailFarmerView: View {
+    
+    @State var selectedFarmer: FarmerPlace?
+    
+    @Binding var show: Bool
+    
+    var body: some View {
+        
+        Text(selectedFarmer?.name ?? "")
```

Comment:
Do you really want to display an empty string? The problem here is because your model is optional.

Reply:
Corrected